mdsuite/visualizer/znvis_visualizer.py

```diff
@@ -130,14 +128,13 @@ def _prepare_species(self):
         return particle_list
 
     def run_visualization(self):
-        # has been removed
         """
         Run the visualization.
 
         Returns
         -------
         Opens the ZnVis app and runs the visualization.
         """
-        # particle_list = self._prepare_species()
-        # visualizer = znvis.Visualizer(particles=particle_list, frame_rate=50)
-        # visualizer.run_visualization()
+        particle_list = self._prepare_species()
+        visualizer = znvis.Visualizer(particles=particle_list, frame_rate=17)
```

Comment:
why is 17 the default?

Reply:
No reason at all, I can change it to 42, I think that is the web js default for videos.

Reply:
For cinema 24 would be the Standard and for TV about 30. I don't think the default ahould be higher than that. But 17 is probably to low, I agree.

Reply:
It depends on what you're trying to see... I will make it an argument for the user.